### ISSUE TITLE
FPWD Request for Contacts Picker API

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/transitions#457]
# Document title, URLs, estimated publication date
Contact Picker API
https://w3c.github.io/contact-api/spec/index.html
Publication date: 2022-10-04

# Abstract
An API to give one-off access to a user’s contact information with full control over the shared data.

# Status
https://w3c.github.io/contact-api/spec/index.html#sotd

(will be updated once we put it into the W3C template)

# Is it a delta specification intended to become a W3C Recommendation?
No

# Link to group's decision to request transition
https://github.com/w3c/contact-api/issues/49

# Information about implementations known to the Working Group
Implemented in Chromium and Safari (behind a flag).

**Comment by plehegar** [comment]
Done under the WebApps WG: https://www.w3.org/2022/04/webapps-wg-charter.html

Approved

**Comment by anssiko** [comment]
@plehegar should https://www.w3.org/Member/wiki/DirectorFOdashboard#Devices_and_Sensors_proposed_charter be resolved prior to this publication to secure wider Call for Exclusions?

**Comment by plehegar** [comment]
@anssiko good point. It's meant to be a joint deliverable indeed but cannot be published as such. Removed the approval to publish for now, and waiting on the FO.

**Comment by marcoscaceres** [comment]
Once the FO is resolved, just as an aside, can we reclaim the https://www.w3.org/TR/contacts-api/ namespace for this? 

And it might be nice to have https://www.w3.org/TR/contacts-manager-api/ maybe also redirect to this new one?

**Comment by siusin** [comment]
@marcoscaceres 
Sure. I'll make these requests when we publish the spec.

**Comment by marcoscaceres** [comment]
@plehegar is there a chance it could we can have it dropped from the DAS Charter? This is now blocking Payment Request too (which depends on Contacts API). 

Everyone in DAS is in WebApps anyway (but not the other way around), so it would be great if WebApps could own it and we could publish it.

**Comment by anssiko** [comment]
>This is now blocking Payment Request too (which depends on Contacts API).

@marcoscaceres are you referring to https://github.com/w3c/payment-request/pull/996 or is there something else? The Contact Picker API reference in that open PR is an informative one. Informative reference stability or Rec Track status has not been a blocker for spec advancement in general. I'm confident we'll get the Contact Picker API on the Rec Track per consensus and this worry will be cleared before it potentially becomes an issue.

>Everyone in DAS is in WebApps

As an aside, this is not true.

**Comment by marcoscaceres** [comment]
Well, the fact that I can't publish both specs is an issue already. Secondly, seems pretty bad to be citing an Unofficial document for a spec that Payments is relying on. Lastly, I can't get the exported definitions from an Unofficial document (nor would I want to).   

> As an aside, this is not true.

Yes it is, apart from:

• "Ecole Mohammadia d'Ingenieurs Rabat (EMI)"
• "Fraunhofer Gesellschaft"

🤷‍♂️

Everyone else is the same:
 
• "Google LLC"
• "Intel Corporation"
• "W3C"
• "Samsung Electronics Co., Ltd."
• "Microsoft Corporation"

**Comment by anssiko** [comment]
@marcoscaceres the [W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20200915/) is not contribution-based, but participation-based. That is, whether someone contributes or not, doesn't matter in this regard.

That said, the DAS WG has worked on the [precursor for this API](https://www.w3.org/standards/history/contacts-api) so also contributions could be expected once this API advances to the Rec Track. I hope soon!

**Comment by marcoscaceres** [comment]
> @marcoscaceres the [W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20200915/) is not contribution-based, but participation-based. That is, whether someone contributes or not, doesn't matter in this regard.

Yes it does. If you have an idle working group putting together bad ideas, then non is going to use the spec, now are they? It sure as does matter. Also, if the participants have not IP in the space or stake in the space, it also matters.

> That said, the DAS WG has worked on the [precursor for this API](https://www.w3.org/standards/history/contacts-api) so also contributions could be expected once this API advances to the Rec Track.

Not sure why that matters. That's ancient history. We could have published this document as CR already if we weren't waiting on DAS.

**Comment by anssiko** [comment]
@marcoscaceres I understand your frustration and aspiration to move faster. It shines through your tone. :heart:

On topic: when I first proposed this API to be brought to the Rec Track in https://github.com/w3c/das-charter/issues/97 I was expecting us to get there faster. However, I also understand there are good reasons a process is in place to govern these transitions.

**Comment by marcoscaceres** [comment]
I don't know... I'm now questioning the logic of having it in DAS at all: it buys us very little and just adds  a lot of bureaucracy for no benefit that I can see. I certainly didn't think it would be such a struggle to get things published so I'd be strongly in favor of dropping it from DAS's Charter. 

The DAS charter expires 2022-11-30, will there be a recharter? (these should be) ... and if so, we should re-evaluate all the stuff that's in the current DAS Charter.

**Comment by marcoscaceres** [comment]
Any update on the council decision? (or are minutes available) wondering if we can get this moving again 🙏

**Comment by plehegar** [comment]
Given the approval of new DAS charter, this transition is approved.

**Comment by anssiko** [comment]
@himorin here's the CfC from the DAS WG:

[CfC: Publish FPWD of Contact Picker API; respond by 9 Dec 2022](https://lists.w3.org/Archives/Public/public-device-apis/2022Dec/0004.html)

I believe 7 days is reasonable.

**Comment by marcoscaceres** [comment]
Ok, we should be good to go here. @himorin, let us know if we can do anything to help!

**Comment by himorin** [comment]
memo: resolution of CfC at
https://lists.w3.org/Archives/Public/public-device-apis/2022Dec/0008.html

**Comment by himorin** [comment]
published: https://www.w3.org/TR/2022/WD-contact-picker-1-20221220/
